src/main/bash/sdkman-install.sh

```diff
@@ -120,11 +120,15 @@ function __sdkman_download() {
 	version="$2"
 
 	archives_folder="${SDKMAN_DIR}/archives"
+	metadata_folder="${SDKMAN_DIR}/var/metadata"
+	mkdir -p ${metadata_folder}
```

Comment:
I think this is fine for now, but the creation of this folder can probably be performed by the install and selfupdate hooks in the future.